examples/tree_2d_dgsem/elixir_navier_stokes_es.jl

```diff
@@ -0,0 +1,75 @@
+using OrdinaryDiffEq
+using Trixi
+
```

Comment:
Similar with this elixir. I mainly added it for debugging / testing purposes. It does not look like it is ever used in the testing.